### ISSUE TITLE
Coaster track not drawn at tunnel exit

### ISSUE THREAD

**Issue by fbourigault** [OpenRCT2/OpenRCT2#7466]
**OS:** macOS (x86-64)
**Version:** v0.1.3-HEAD
**Commit/Build:** 3025599

The Dynamite blaster track is not drawn at tunnel exit just before the lift hill.

The bug also happen in v0.1.2.

**Steps to reproduce:**
1. Start a new game using Dynamites Dune scenario.

**Screenshots / Video:**
![dynamite dunes 2018-04-30 16-14-43](https://user-images.githubusercontent.com/1116116/39431888-6003976e-4c92-11e8-9e2f-fd82bceb1001.png)

**Comment by jensj12** [comment]
I think this is an original bug, can't check though. It happens because of the helix.

**Comment by RCTMASTA** [comment]
definitely remember seeing this in vanilla, yeah.

**Comment by fbourigault** [comment]
It's an original bug :
![scr1](https://user-images.githubusercontent.com/1116116/39520494-23ee4e5c-4e0b-11e8-93fc-b343e4b88cf6.jpg)

**Comment by karst** [comment]
This seems to occur quite often. I noticed this while working on adding inline twists to several coaster types and this didnt have anything to do with the code I wrote. 
![image](https://user-images.githubusercontent.com/1516893/149635441-f63bc831-f998-403b-9778-fbad8c5fa1fd.png)

Interestingly enough this bug occurs in RCT Classic aswell, but it doesn't in the original RCT.
![image](https://user-images.githubusercontent.com/1516893/149635451-7f9fabd5-3a3c-4889-9221-5901dba2df70.png)
(RCT Classic)

![image](https://user-images.githubusercontent.com/1516893/149635458-cb7d5065-0dee-4b35-aa86-b9a427333172.png)
(RCT)
